src/ObjectUtils/include/ObjectUtils/ObjectFile.h

```diff
@@ -44,7 +44,14 @@ class ObjectFile : public SymbolsFile {
   // The same calculation applies for PE/COFF object files. Here, the address
   // to be subtracted is ImageBase as specified in the PE header.
   [[nodiscard]] virtual uint64_t GetLoadBias() const = 0;
+
+  // This is the offset from the base address of the text segment *when loaded into memory*. Note
```

Comment:
Maybe I'm reading this sentence wrong, but "This is the offset of the text segment *when loaded into memory* relative to the base address." might be clearer.